.github/workflows/deploy.yml

```diff
@@ -1,6 +1,7 @@
 name: Deploy
 
 on:
+  # Remove this before committing to main
```

Comment:
what needs to be removed?

Reply:
I had changed the triggers to let this run on commit.  If you go through the commit history you should see it.